### ISSUE TITLE
Error with creating album aborts following album creations

### ISSUE THREAD

**Issue by pgebheim** [simulot/immich-go#61]
## Invocation
```
% ./immich-go -server=http://immich:30041 -key=$IMMICH_KEY upload -create-albums -keep-untitled-albums -google-photos takeout-*.zip
immich-go  0.8.0, commit b1fcb87396ef4789372637724219d6cea54389ce, built at 2023-11-04T20:47:22Z
```

## Symptom
After uploading images from my takeout archives, immich got to the "Managing Albums" step. At this point it tried to create an album with an empty name causing an error to be thrown. This error aborted further album management tasks.

### Logs
```
Create the album
can't create the album list from the server: CreateAlbum, POST, http://immich:30041/api/album, 400 Bad Request
Bad Request
albumName should not be empty

2 server assets to delete.
41439 media scanned, 39813 uploaded.
Done.
```

## Expected behavior
1. Immich should filter out known bad / empty album names.
2. Error handling should be inside the per-album creation loop, so a single album error doesn't stop the whole process.

## Further question
If I patch this and run it again, will it successfully create the albums without re-uploading the images?

**Comment by pgebheim** [comment]
I would expect this line to continue instead of return

https://github.com/simulot/immich-go/blob/b1fcb87396ef4789372637724219d6cea54389ce/cmdupload/upload.go#L592C7-L592C7

**Comment by simulot** [comment]
Thank you for reporting your case.
I wasn't able to reproduce it with same flags and with some weird google photos albums.

Coud you share the metadata.json file corresponding to the album? and the takeout folder structure?
You can DM me at my discord account @ simulot  


And, yes, albums will be created without re-uploading the photos.

**Comment by simulot** [comment]
A temporary fix is available with 0.8.1

**Comment by pgebheim** [comment]
Thanks so much! Sorry I haven't gotten a a chance to send you the details, I'll do it this weekend.
